project/src/components/map/map.tsx

```diff
@@ -4,14 +4,20 @@ import 'leaflet/dist/leaflet.css';
 import useMap from '../../hooks/useMap';
 import {ICON_ANCHOR_HEIGHT, ICON_ANCHOR_WIDTH, ICON_SIZE, URL_MARKER_CURRENT, URL_MARKER_DEFAULT} from '../../const';
 import {CityType, OfferType} from '../../types/mocksTypes';
+import {PageType} from '../../types/propsTypes';
 
 type MapType = {
   city: CityType,
   offers: OfferType[],
-  activeId?: string
+  activeId?: string,
+  pageType?: PageType,
 };
 
-function Map({city, offers, activeId}: MapType): JSX.Element {
+function getMapHeight(pageType: PageType | undefined): string {
```

Comment:
```suggestion
function getMapHeight(pageType?: PageType): string {
```
Зачем явно передавать undefined? Ну и тут прекрасно стрелочная функция в одну строку заходит)